From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Sun, 24 May 2020 19:38:18 +0200
Subject: [PATCH] fix building with gcc 10

Without this, building with gcc-10 fails with:

/usr/bin/ld: CMakeFiles/genisoimage.dir/apple.o:.../genisoimage/geniso
image.h:379: multiple definition of `outfile'; CMakeFiles/genisoimage.dir/genisoimage.o:.../genisoimage/genisoimage.h:379: first defined here

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 genisoimage/genisoimage.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/genisoimage/genisoimage.h b/genisoimage/genisoimage.h
index bbedfb08da6a..82c859bc93e8 100644
--- a/genisoimage/genisoimage.h
+++ b/genisoimage/genisoimage.h
@@ -376,7 +376,7 @@ extern int	use_fileversion;
 extern int	split_SL_component;
 extern int	split_SL_field;
 extern char	*trans_tbl;
-char		*outfile;
+extern char	*outfile;
 
 #define	JMAX		64	/* maximum Joliet file name length (spec) */
 #define	JLONGMAX	103	/* out of spec Joliet file name length */
